tests/integ/conftest.py

```diff
@@ -329,7 +329,7 @@ def _notify(self, query_record: QueryRecord, *args, **kwargs) -> None:
 # Disable phase0 Snowpark IR check by setting autouse=False,
 # by default enabled for all tests (except modin/) to check that Snowpark python requests contain
 # the dataframeAst property holding the encoded representation.
```

Comment:
this will need to get fixed in another PR. deactivating for now, so the integ tests are run.